api/v01/entities/vrp_result.rb

```diff
@@ -56,71 +53,83 @@ class VRPResultDetailedCosts < Grape::Entity
 
     class VrpResultSolutionRouteActivities < Grape::Entity
       expose :point_id, documentation: { type: String, desc: 'Linked spatial point' }
-      expose :travel_distance, documentation: { type: Integer, desc: 'travel distance from the previous point' }
-      expose :travel_duration, documentation: { type: Integer, desc: 'travel time from the previous point' }
-      expose :waiting_duration, documentation: { type: Integer, desc: '' }
-      expose :arrival_time, documentation: { type: Integer, desc: '' }
+      expose :travel_distance, documentation: { type: Integer, desc: 'Travel distance from previous point' }
+      expose :travel_time, documentation: { type: Integer, desc: 'Travel time from previous point' }
+      expose :travel_value, documentation: { type: Integer, desc: 'Travel value from previous point' }
+      expose :waiting_time, documentation: { type: Integer, desc: '' }
+      expose :begin_time, documentation: { type: Integer, desc: 'Time visit starts' }
+      expose :end_time, documentation: { type: Integer, desc: 'Time visit ends' }
       expose :departure_time, documentation: { type: Integer, desc: '' }
```

Comment:
do we want to keep `departure_time`? I thought we replace arrival/departure with begin/end, is it not?

Reply:
It was initially removed but @braktar says this can have an impact for Keesing when they generate their classes.